toolchain-analysis/bazel.md

```diff
@@ -0,0 +1,41 @@
+# Bazel toolchain analysis
+
+[Bazel](https://bazel.build/) is an open-source build and test tool similar to
+Make, Maven, and Gradle. It uses a human-readable, high-level build language.
+Bazel supports projects in multiple languages and builds outputs for multiple
+platforms. Bazel supports large codebases across multiple repositories, and
+large numbers of users.
+
+## Supported platforms
+
+TODO: test Bazel against currently Node.js supported platforms and toolchains
+(see [BUILDING.MD](https://github.com/nodejs/node/edit/master/BUILDING.md)).
+
+### Unsupported platforms
+
+TODO: see above
+
+## Pros
+
+* Easier to upgrade V8 as we don't need to rewrite build configs for V8
+* Compared to GN, Bazel is more widely adopted outside Chromium
+
+## Cons
+
+* Other dependencies will need Node.js-maintained Bazel build files
+* Currently support a limited set of platforms
```

Comment:
I'm curious which host platforms node supports and Bazel doesn't? Perhaps the Bazel team can ship more binaries.

Reply:
It's less about bazel running on the platform and more about V8 providing the configurations for those platforms or not (according to [this comment](https://github.com/nodejs/TSC/issues/901#issuecomment-953641136) the range of platforms supported by the bazel configs is only a subset of currently supported platforms with GN).